test/test_config.py

```diff
@@ -1,17 +1,57 @@
 import os.path
 import tempfile
-import unittest
+from unittest import TestCase
 from pathlib import Path
-
 from config import AlltalkConfig, AlltalkTTSEnginesConfig, AlltalkNewEnginesConfig
 
 
-class TestAlltalkConfig(unittest.TestCase):
+class TestAlltalkConfig(TestCase):
 
     def setUp(self):
         self.config = AlltalkConfig.get_instance()
         self.config.reload()
 
+    def test_default_values_loaded(self):
+        cfg = AlltalkConfig(Path(__file__).parent.resolve() / 'empty.json')
+
+        # Since the default config file is expected to also contain the
+        # default values, we can simply check that both dictionaries are identical:
+        self.assertEqual(self.config.to_dict(), cfg.to_dict())
+
+    def test_no_default_values_missing(self):
```

Comment:
Check if you want this: This forces you to update `confignew.json` once you add new config properties in code. I personally think this is a good idea, but you might not...